src/models/accommodationtype.js

```diff
@@ -0,0 +1,12 @@
+export default (sequelize, DataTypes) => {
+  const AccommodationType = sequelize.define('AccommodationType', {
+    name: DataTypes.STRING,
+    category: DataTypes.STRING,
+    stars: DataTypes.INTEGER,
+    locationId: DataTypes.INTEGER,
+    peopleCapacity: DataTypes.INTEGER,
+    rooms: DataTypes.INTEGER,
+    availability: DataTypes.BOOLEAN
```

Comment:
A variable that holds a boolean value must should be prefixed with either `is` or `has`.

Reply:
@adesege  can you please give more clarification on this? I'm not understanding it

Reply:
If a variable is going to hold a boolean data type, it should be prefixed with `is` or `has`. For example, `isLoading`, `hasAge` etc